tests/oeedger8r/edl/deepcopy.edl

```diff
@@ -117,5 +149,82 @@ enclave {
 
     // Test a real-world scenario.
     public void deepcopy_iovec([in, out, count=n] IOVEC* iov, size_t n);
+
+    // Expect the callee to set a large count value on return.
+    public void deepcopy_countparam_return_large([in, out, count=1] CountParamStruct* s);
```

Comment:
Add tests for switchless calls as well.
One way you could avoid duplication is to make all these calls switchless and then test the switchless calls.
Then create another enclave without setting the switchless configuration parameter so that these calls will be routed normally.

Reply:
Done